src/core/MOM_open_boundary.F90

```diff
@@ -1962,6 +1962,7 @@ subroutine radiation_open_bdry_conds(OBC, u_new, u_old, v_new, v_old, G, US, dt)
            if (dhdt*dhdx < 0.0) dhdt = 0.0
            rx_new = dhdt*dhdx
            cff_new = max(dhdx*dhdx + dhdy*dhdy, eps)
+           rx_new = min(rx_new, cff_new*rx_max)
```

Comment:
These changes look good to me, but a slightly more parsimonious and bitwise equivalent approach would be to merge line 1963 into the first argument of line 1965, and similarly for the other analogous 3 cases later in the code.